### PR TITLE
TWT-1055 FAQ repository updates

### PR DESCRIPTION
**Pull request by Barshad96**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ Select [<mark style="color:blue;">**Try now**</mark>](https://cloud.ionos.com/co
...
 | [<mark style="color:blue;">Can I limit my costs?](general-information/frequently-asked-questions/costs-and-billing.md#can-i-limit-my-costs)                                  | [<mark style="color:blue;">How do I manage user permissions?</mark>](general-information/frequently-asked-questions/data-center-designer.md#how-do-i-manage-user-permissions) |
-| [<mark style="color:blue;">Can I calculate my costs in advance?</mark>](general-information/frequently-asked-questions/costs-and-billing.md#can-i-calculate-my-expenses-in-advance) | [<mark style="color:blue;">How are user roles assigned?</mark>](general-information/frequently-asked-questions/data-center-designer.md#how-are-user-roles-assigned)           |
+| [<mark style="color:blue;">Can I calculate my costs in advance?</mark>](general-information/frequently-asked-questions/costs-and-billing.md#can-i-calculate-my-expenses-in-advance) | [<mark style="color:blue;">How are user roles assigned?</mark>](general-information/frequently-asked-questions/data-center-designer.md#how-are-user-privileges-assigned)           |
```
Comment (DiPoojary) [inline README.md:24]: How are user **roles** assigned? => How are user **privileges** assigned?

File: general-information/frequently-asked-questions/account-settings.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 
 ### Do I need a verification code to log in to IONOS Cloud account?
 
-No verification code is required to log in to your IONOS Cloud account for the first time. Once the **2-Step Verification** is **activated**, you cannot log in again without the code. If the **Authenticator** app is deleted or the device it's on is not available you will need to contact the support team and request to turn off the **2-Step Verification**. You can then log in again and reactivate the **2-Step Verification** yourself.
+No verification code is required to log in to your IONOS Cloud account for the first time. Once the **2-Step Verification** is **activated**, you cannot log in again without the code. If the **Authenticator** app is deleted or the device it's on is not available, you will need to contact the support team and request to turn off the **2-Step Verification**. You can then log in again and reactivate the **2-Step Verification** yourself.
```
Comment (DiPoojary) [inline general-information/frequently-asked-questions/account-settings.md:13]: **app** is deleted or **the device it's on is not available,** => **application** is deleted or the device is not available,

File: general-information/frequently-asked-questions/compute-engine.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
+The cloud server is suitable for anyone who wants a robust IT infrastructure at a reasonable price. It is the perfect foundation for various web projects, whether it is a private homepage or company website, a robust framework for web developers, or a high-performance gaming server. With the cloud servers, you can flexibly balance the necessary resources and pay only for what is actually used. If you are working on larger projects in which you use servers for different purposes, such as a front-end server with a load balancer in conjunction with database servers that you manage from third-party software via an API, then you can easily match such scenarios with the cloud server solutions. You also get unlimited traffic and high availability.
```
Comment (DiPoojary) [inline general-information/frequently-asked-questions/compute-engine.md:5]: With **the** cloud servers, =>With cloud servers,

The following sentence is too big:

 If you are working on larger projects in which you use servers for different purposes, such as a front-end server with a load balancer in conjunction with database servers that you manage from third-party software via an API, then you can easily match such scenarios with the cloud server solutions.

File: general-information/frequently-asked-questions/compute-engine.md [hunk 4]
```diff
@@ -2,11 +2,11 @@
...
 
-Yes, the cloud plans provided by IONOS are ideal for small and medium businesses. Just choose the appropriate tariff for your needs based on the required server resources. They are usually differentiated by Central Processing Unit (CPU), Random Access Memory (RAM), and memory capacity. Fast Solid State Disk (SSD) with low access times is available in every plan for data transfer and storage. Whenever you need more resources, you can adjust them in real time as needed. For larger companies that need more performance and server resources, we recommend using **Compute Engine**.
+Yes, the cloud plans provided by IONOS are ideal for small and medium businesses. Choose the appropriate tariff for your business needs based on the required server resources. They are usually differentiated by Central Processing Unit (CPU), Random Access Memory (RAM), and memory capacity. Fast Solid State Disk (SSD) with low access times is available in every data transfer and storage plan. Whenever you need more resources, you can adjust them in real time as needed. For larger companies that need more performance and server resources, we recommend using **Compute Engine**.
```
Comment (DiPoojary) [inline general-information/frequently-asked-questions/compute-engine.md:9]: Whenever you need more resources, you can adjust them in real time as needed. => You can adjust resources in real-time as needed.

For larger companies that need more performance and server resources, we recommend using **Compute Engine**. => We recommend using **Compute Engine** for larger companies needing more performance and server resources.

File: README.md (hunk 1 above)
Comment (Barshad96) [reply README.md]: Done, thank you.

File: general-information/frequently-asked-questions/account-settings.md (hunk 2 above)
Comment (Barshad96) [reply general-information/frequently-asked-questions/account-settings.md:13]: Done, thank you.

File: general-information/frequently-asked-questions/compute-engine.md (hunk 3 above)
Comment (Barshad96) [reply general-information/frequently-asked-questions/compute-engine.md:5]: Done, thank you :)

File: general-information/frequently-asked-questions/compute-engine.md (hunk 4 above)
Comment (Barshad96) [reply general-information/frequently-asked-questions/compute-engine.md:9]: Thank you so much, updating it to:  You can adjust these resources in real-time as needed.
Updated the second as well.

File: general-information/frequently-asked-questions/compute-engine.md [hunk 5]
```diff
@@ -2,11 +2,12 @@
...
+The cloud server is suitable for anyone who wants a robust IT infrastructure at a reasonable price. It is the perfect foundation for various web projects, whether it is a private homepage or company website, a robust framework for web developers, or a high-performance gaming server. With cloud servers, you can flexibly balance the necessary resources and pay only for what is actually used. 
+If you are working on larger projects, you can use servers for various purposes. For example, you have a front-end server paired with a load balancer, along with database servers managed through third-party software using an API. In such cases, cloud server solutions offer a seamless match to address these diverse scenarios.
```
Comment (DiPoojary) [inline general-information/frequently-asked-questions/compute-engine.md:6]: For example, **consider** you have a front-end server paired

Comment (Barshad96) [reply general-information/frequently-asked-questions/compute-engine.md]: Done, thank you so much

File: general-information/frequently-asked-questions/general-inquiries.md [hunk 6]
```diff
@@ -12,23 +12,23 @@ If you require direct assistance, contact [<mark style="color:blue;">IONOS Cloud
 
 ### If I need a certain feature but cannot find it on the website, how do I know if IONOS Cloud offers it?
 
-If you already have an account with us, get in touch with your account manager. If you do not have an account with us, contact us using [<mark style="color:blue;">IONOS Cloud help</mark>](https://cloud.ionos.com/help).
+If you already have an account with us, contact your account manager. If you do not have an account with us, contact us using [<mark style="color:blue;">IONOS Cloud help</mark>](https://cloud.ionos.com/help).
```
Comment (DiPoojary) [inline general-information/frequently-asked-questions/general-inquiries.md:15]: If you already have an account with IONOS, contact your account manager. If not, you can contact the [<mark style="color:blue;">IONOS Cloud help</mark>](https://cloud.ionos.com/help).

File: general-information/frequently-asked-questions/general-inquiries.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
 ### How can I ensure my data center is properly set up?
 
-Customers should have a basic understanding of cloud hosting platforms before using IONOS's Cloud Products and services. The best way to get started is by using IONOS graphic user interface tutorials. These instruction sets will teach you how to launch a basic data center and set up initial user access and permissions.
+Customers should have a basic understanding of cloud hosting platforms before using IONOS's Cloud products and services. The best way to get started is by using IONOS graphic user interface tutorials. These instruction sets will teach you how to launch a fundamental data center and set up initial user access and permissions.
```
[the remark this answers is not in the document]
Comment (Barshad96) [reply general-information/frequently-asked-questions/general-inquiries.md]: Done, thank you

File: general-information/frequently-asked-questions/general-inquiries.md [hunk 8]
```diff
@@ -12,23 +12,23 @@ If you require direct assistance, contact [<mark style="color:blue;">IONOS Cloud
...
 
 ### Can I get help in setting up my Virtual Data Center?
 
 We have extensive [<mark style="color:blue;">IONOS Cloud Documentation</mark>](https://docs.ionos.com/cloud/) and offer many informative [<mark style="color:blue;">webinars</mark>](https://cloud.ionos.com/webinars). You can also customize your own infrastructure according to your needs. If you need IT service provider support, we would be happy to introduce you to a partner.
 
 ### Does IONOS Cloud offer specific certifications?
 
-You can find IONOS [<mark style="color:blue;">Certificates and attestations</mark>](https://cloud.ionos.de/zertifikate). If you need additional certificates, reach out to us using [<mark style="color:blue">IONOS customer service</mark>](https://contact.ionos.de/) or get in touch with your account manager.
+You can find IONOS [<mark style="color:blue;">Certificates and attestations</mark>](https://cloud.ionos.de/zertifikate). If you need additional certificates, contact us using [<mark style="color:blue">IONOS customer service</mark>](https://contact.ionos.de/) or contact your account manager.
```
Comment (DiPoojary) [inline general-information/frequently-asked-questions/general-inquiries.md:27]: Yes, IONOS offers specific certifications. For more information, see` [<mark style="color:blue;">Certificates and attestations</mark>](https://cloud.ionos.de/zertifikate)`. If you need additional certificates, you can either contact `[<mark style="color:blue">IONOS customer service</mark>](https://contact.ionos.de/)` or contact your account manager.

**Review by DiPoojary — APPROVED** [review]

**Pull request merged**